web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -489,6 +493,79 @@ describe("Formation - BusinessStep", () => {
         expect(screen.queryByText(Config.formation.fields.willPracticeLaw.label)).not.toBeInTheDocument();
       });
     }
+
+    describe("Business Designator Options based on Will Practice Law Answer", () => {
+      it.each(corporateLegalStructures)(
+        "Shows PA and PC options for Business Designator when Will You Practice Law is Yes",
+        async (legalStructureId) => {
+          await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, { willPracticeLaw: true });
+
+          expect(screen.queryByText("P.C.")).not.toBeInTheDocument();
+          expect(screen.queryByText("P.A.")).not.toBeInTheDocument();
+
+          await userEvent.click(screen.getByTestId("business-suffix-main"));
+          expect(screen.getByText("P.C.")).toBeInTheDocument();
+          expect(screen.getByText("P.A.")).toBeInTheDocument();
+        }
+      );
+
+      it.each(corporateLegalStructures)(
+        "Does not show PA and PC options for Business Designator when Will You Practice Law is No",
+        async (legalStructureId) => {
+          await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, { willPracticeLaw: false });
+          expect(screen.queryByText("P.C.")).not.toBeInTheDocument();
+          expect(screen.queryByText("P.A.")).not.toBeInTheDocument();
+
+          await userEvent.click(screen.getByTestId("business-suffix-main"));
```

Comment:
[sand] can we rename this into a function? it's kinda difficult to tell what this click is trying to do?